portfolio/src/main/webapp/script.js

```diff
@@ -194,6 +189,10 @@ function getUserLocation() {
         });
         marker.setMap(map);
         map.setCenter(userLocation);
+
+        var circle = new google.maps.Circle(
```

Comment:
let instead of var

also rename `circle` to something like `locationLimitCircle` so it's clearer

Reply:
Sounds good, thank you!